atlp-rwanda/hackers-ec-Fe
# Enhancement: Reorganize Test Directory to Align with src folder Structure

#### How should this be manually tested?
1. Clone the repository.
2. Check out to the branch **enhance-setup-testing**
3. Run **npm install** to install dependencies.
4. Run **npm run test** to execute tests using Vitest.
5. Verify that all tests pass and coverage is reported.
